ext/session/session.stub.php

```diff
@@ -43,35 +43,45 @@ function session_commit(): bool {}
  * @param callable|object $open
  * @param callable|bool $close
  */
-function session_set_save_handler($open, $close = UNKNOWN, callable $read = UNKNOWN, callable $write = UNKNOWN, callable $destroy = UNKNOWN, callable $gc = UNKNOWN, callable $create_sid = UNKNOWN, callable $validate_sid = UNKNOWN, callable $update_timestamp = UNKNOWN): bool {}
+function session_set_save_handler(
+    $open,
+    $close = UNKNOWN,
+    callable $read = UNKNOWN,
+    callable $write = UNKNOWN,
+    callable $destroy = UNKNOWN,
+    callable $gc = UNKNOWN,
+    callable $create_session_id = UNKNOWN,
+    callable $validate_session_id = UNKNOWN,
+    callable $update_timestamp = UNKNOWN
+): bool {}
 
 function session_cache_limiter(?string $cache_limiter = null): string|false {}
 
 function session_cache_expire(?int $new_cache_expire = null): int|false {}
```

Comment:
Hm, I'm not sure what our convention for these kinds of functions is. But at the least the `new_` prefix here vs the previous function is inconsistent.

Reply:
I think we could just use `$value` in these 2 functions if you agree. I see no reason to repeat what the function name already contains.

Reply:
I'm also ok to just remove the `new_` prefix, it's inconsistent for sure.

Reply:
Just `$value` looks reasonable.